theano/sandbox/gpuarray/tests/test_dnn.py

```diff
@@ -278,24 +278,31 @@ def test_pooling_opt():
     if not dnn.dnn_available():
         raise SkipTest(dnn.dnn_available.msg)
 
-    x = T.ftensor4()
+    x = T.matrix()
```

Comment:
The test currently fail, as it need to run with floatX=float32 and floatX=float64. Just change that line to T.fmatrix() and it is good to merge.

thanks